packages/dashboard/src/components/Wizard/Wizard.tsx

```diff
@@ -38,3 +38,10 @@ export class Wizard extends React.Component<IWizardProps, {}> {
     );
   }
 }
+
+export const Wizard: (props: IWizardProps) => JSX.Element = styled<IWizardProps, IWizardStyleProps, IWizardStyles>(
```

Comment:
Can we put the base and Wizard in separate files to be consistent with rest of Fabric?

Reply:
fixed